client/apps/settings/client/curations/display/components/canvas_images.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { Col, Row } from 'react-styled-flexboxgrid'
+import ImageUpload from 'client/apps/edit/components/admin/components/image_upload.coffee'
+
+export default class CanvasImages extends React.Component {
```

Comment:
Not sure if you saw [this convo](https://artsy.slack.com/archives/C2TQ4PT8R/p1507801960000308) (and it's definitely not necessary for this PR) but it brings up good points about exporting with `default`!